src/Microsoft.OData.Client/Serialization/HttpWebRequestMessage.cs

```diff
@@ -187,6 +187,15 @@ public override int Timeout
             set { this.httpRequest.Timeout = (int)Math.Min(Int32.MaxValue, new TimeSpan(0, 0, value).TotalMilliseconds); }
         }
 
+        /// <summary>
+        /// Gets or sets the read and write timeout (in seconds) for this request.
+        /// </summary>
+        public override int ReadWriteTimeout
+        {
+            get { return this.httpRequest.Timeout; }
+            set { this.httpRequest.Timeout = (int)Math.Min(Int32.MaxValue, new TimeSpan(0, 0, value).TotalMilliseconds); }

```

Comment:
```suggestion
            get { return this.Timeout; }
            set { this.Timeout = value); }
```
Seems to be repeating the Timeout code, might as well just forward these requests to that.

Reply:
fixed